qa/tasks/scripts/munge_policy.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# munge_policy.sh
+#
+# Adds or removes roles from policy.cfg
+#
+
+set -ex
+
+OPERATION=$1 # add_node_as_role / remove_storage_only_node
+NODE_FQDN="${2}" # target192168000063.teuthology
+ROLE="${3}" # storage
+POLICY_CFG="${4}"
```

Comment:
@shyukri Please change the line `OPERATION=$1` so it's in the same coding style as the subsequent lines

Reply:
Done